timemachine/md/minimizer.py

```diff
@@ -59,15 +59,18 @@ def parameterize_system(topo, ff: Forcefield, lamb: float):
             lamb,
         ),
     ]
-    return params_potential_pairs
+    return [p for _, p in params_potential_pairs], [params for params, _ in params_potential_pairs]
 
 
-def bind_potentials(params_potential_pairs):
-    pots = [pot for _, pot in params_potential_pairs]
-    params = [p for p, _ in params_potential_pairs]
-    flat_params = np.concatenate([p.reshape(-1) for p in params])
-    u_impls = [SummedPotential(pots, params).bind(flat_params).to_gpu(precision=np.float32).bound_impl]
-    return u_impls
+def flatten_params(params: List[NDArray]) -> NDArray:
+    return np.concatenate([p.reshape(-1) for p in params])
+
+
+def summed_potential_from_potentials_and_params(
+    potentials: List[Potential], params: List[NDArray]
+) -> custom_ops.BoundPotential:
+    flat_params = flatten_params(params)
+    return SummedPotential(potentials, params).bind(flat_params).to_gpu(precision=np.float32).bound_impl
```

Comment:
nit: naming inconsistency -- rename to clunkier `summed_potential_bound_impl_from_potentials_and_params` or remove `.bound_impl` here?

Reply:
Adopted the clunkier name to be explicit